tests/integration/targets/docker_config/tasks/test_docker_config.yml

```diff
@@ -226,6 +227,100 @@
       that:
         - output.failed
 
+# template_driver tests
```

Comment:
Put these into a `block:` with a `when:` condition that only matches when the feature is available.